test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE resultCode to Navigation.AlertManeuver component
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session are started
+-- 3) App is registered
+-- 4) App is activated
+--
+-- Steps:
+-- 1) Send AlertManeuver mobile RPC from app with parameter imageType = "STATIC", HMI responds with
+--    UNSUPPORTED_RESOURCE to Navigation.AlertManeuver request
+--
+-- SDL does:
+-- 1) Respond with 'AlertManeuver', { success = true, resultCode = "UNSUPPORTED_RESOURCE" } to App
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local params = {
+  ttsChunks = {
+    {
+      type = "TEXT",
+      text = "Text to Speak"
+    }
+  },
+  softButtons = {
+    {
+      systemAction = "DEFAULT_ACTION",
+      isHighlighted = true,
+      softButtonID = 5517,
+      type = "BOTH",
+      image = {
+        value = "icon.png",
+        imageType = "STATIC"
+      },
+      text = "Close"
+    }
+  }
+}
+
+--[[ Local Functions ]]
+local function sendAlertManeuver()
+  local mobileSession = common.mobile.getSession()
+  local hmi = common.hmi.getConnection()
+  local dataToHMI = {
+    softButtons = params["softButtons"],
```

Comment:
Here also can be written like
```suggestion
    softButtons = params.softButtons,
```